### ISSUE TITLE
Over zealous automatic bans

### ISSUE THREAD

**Issue by neofight78** [language-learners/phpbb#26]
I keep getting banned for the sin of being in Russia. It would be good if long time users didn't get automatic bans, which are presumably based purely on IP.

Current message:

You have been permanently banned from this board.

Please contact the Board Administrator for more information.

A ban has been issued on your IP address.

**Comment by emk** [comment]
Was this the accidental ban of our nginx reverse proxy that affected everyone? Or is this a separate ban?

If it's the nginx ban, we need to fix phpbb to honor `x-forwarded-for` headers.

**Comment by neofight78** [comment]
Yes, it was. Although there are periods where I do get locked out because my IP has been blacklisted by Spamhaus. Whether it's the blacklist or my IP that keeps changing I don't know.
